website/redirects.js

```diff
@@ -137,6 +137,60 @@ module.exports = [
     destination: '/consul/docs/ecs/reference/compatibility',
     permanent: true,
   },
+  {
+    source: '/consul/docs/connect/gateways/api-gateway/usage',
+    destination:
+      '/consul/docs/connect/gateways/api-gateway/deploy/listeners-vms',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/agi-gateway',
+    destination: '/consul/docs/connect/gateways/agi-gateway',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/install',
+    destination: '/consul/docs/connect/gateways/api-gateway/deploy/install-k8s',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/usage/reroute-http-requests',
+    destination:
+      '/consul/docs/connect/gateways/api-gateway/define-routes/reroute-http-requests',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/usage/route-to-peered-services',
+    destination:
+      '/consul/docs/connect/gateways/api-gateway/route-to-peered-services',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/usage/errors',
+    destination: '/consul/docs/connect/gateways/api-gateway/errors',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/usage/usage',
+    destination:
+      '/consul/docs/connect/gateways/api-gateway/deploy/listeners-k8s',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/upgrades',
+    destination: '/consul/docs/connect/gateways/api-gateway/upgrades-k8s',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/configuration/',
+    destination: '/consul/docs/connect/gateways/api-gateway/configuration/',
+    permanent: true,
+  },
+  {
+    source: '/consul/docs/api-gateway/configuration/:slug',
+    destination:
+      '/consul/docs/connect/gateways/api-gateway/configuration/:slug',
+  },
```

Comment:
Looks like Vercel is failing because this one doesn't include the `permanent` field